### DISCUSSION TITLE
Random Gas Readings

### DISCUSSION THREAD

**Discussion by anothermort** [BottlecapDave/HomeAssistant-OctopusEnergy#868] [Q&A] [answered]
Hi All,

Had this at least a few times now. Gas is only set to heat hot water at 5:30 am. I  do use it to cook.
But keep getting a reading between 00:00 and 01:00 for Gas on the energy tab in HA.

Nothing on Octopus or the meter data shows a reading for gas only 5:30am onwards. Seems to have started this month nothing last month.
Not happening everyday, possibly the day after HA restarted?

Is it just me?

14th May
![gas](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/13786570/d53fe423-772e-4744-8963-903458e90d38)
11th May
![gas2](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/13786570/a0eb3ef7-518b-45b4-baf6-b228083ddea7)

**Comment by BottlecapDave** [comment]
Hello. Do you see these readings on the OE website or app? Is this using the previous consumption sensor or current consumption sensor?

**Reply by anothermort** [reply]
No don't see the readings on the website or the app.

Using the current consumption sensor via the mini.

**Reply by anothermort** [reply]
May 11th 
Consumption (mÂ³) | Start | End
-- | -- | --
0 | 2024-05-11T00:00:00+01:00 | 2024-05-11T00:30:00+01:00
0 | 2024-05-11T00:30:00+01:00 | 2024-05-11T01:00:00+01:00
0 | 2024-05-11T01:00:00+01:00 | 2024-05-11T01:30:00+01:00
0 | 2024-05-11T01:30:00+01:00 | 2024-05-11T02:00:00+01:00
0 | 2024-05-11T02:00:00+01:00 | 2024-05-11T02:30:00+01:00
0 | 2024-05-11T02:30:00+01:00 | 2024-05-11T03:00:00+01:00
0 | 2024-05-11T03:00:00+01:00 | 2024-05-11T03:30:00+01:00
0 | 2024-05-11T03:30:00+01:00 | 2024-05-11T04:00:00+01:00
0 | 2024-05-11T04:00:00+01:00 | 2024-05-11T04:30:00+01:00
0 | 2024-05-11T04:30:00+01:00 | 2024-05-11T05:00:00+01:00
0 | 2024-05-11T05:00:00+01:00 | 2024-05-11T05:30:00+01:00
0.231 | 2024-05-11T05:30:00+01:00 | 2024-05-11T06:00:00+01:00
0.094 | 2024-05-11T06:00:00+01:00 | 2024-05-11T06:30:00+01:00
0 | 2024-05-11T06:30:00+01:00 | 2024-05-11T07:00:00+01:00
0 | 2024-05-11T07:00:00+01:00 | 2024-05-11T07:30:00+01:00
0 | 2024-05-11T07:30:00+01:00 | 2024-05-11T08:00:00+01:00
0 | 2024-05-11T08:00:00+01:00 | 2024-05-11T08:30:00+01:00
0.045 | 2024-05-11T08:30:00+01:00 | 2024-05-11T09:00:00+01:00
0.043 | 2024-05-11T09:00:00+01:00 | 2024-05-11T09:30:00+01:00
0.003 | 2024-05-11T09:30:00+01:00 | 2024-05-11T10:00:00+01:00


</body>

</html>

**Reply by anothermort** [reply]
10th May in case value had rolled over, but doesn't look like it.
Consumption (mÂ³) | Start | End
-- | -- | --
0 | 2024-05-10T00:00:00+01:00 | 2024-05-10T00:30:00+01:00
0 | 2024-05-10T00:30:00+01:00 | 2024-05-10T01:00:00+01:00
0 | 2024-05-10T01:00:00+01:00 | 2024-05-10T01:30:00+01:00
0 | 2024-05-10T01:30:00+01:00 | 2024-05-10T02:00:00+01:00
0 | 2024-05-10T02:00:00+01:00 | 2024-05-10T02:30:00+01:00
0 | 2024-05-10T02:30:00+01:00 | 2024-05-10T03:00:00+01:00
0 | 2024-05-10T03:00:00+01:00 | 2024-05-10T03:30:00+01:00
0 | 2024-05-10T03:30:00+01:00 | 2024-05-10T04:00:00+01:00
0 | 2024-05-10T04:00:00+01:00 | 2024-05-10T04:30:00+01:00
0 | 2024-05-10T04:30:00+01:00 | 2024-05-10T05:00:00+01:00
0 | 2024-05-10T05:00:00+01:00 | 2024-05-10T05:30:00+01:00
0.309 | 2024-05-10T05:30:00+01:00 | 2024-05-10T06:00:00+01:00
0.137 | 2024-05-10T06:00:00+01:00 | 2024-05-10T06:30:00+01:00
0 | 2024-05-10T06:30:00+01:00 | 2024-05-10T07:00:00+01:00
0 | 2024-05-10T07:00:00+01:00 | 2024-05-10T07:30:00+01:00
0 | 2024-05-10T07:30:00+01:00 | 2024-05-10T08:00:00+01:00
0 | 2024-05-10T08:00:00+01:00 | 2024-05-10T08:30:00+01:00
0 | 2024-05-10T08:30:00+01:00 | 2024-05-10T09:00:00+01:00
0 | 2024-05-10T09:00:00+01:00 | 2024-05-10T09:30:00+01:00
0 | 2024-05-10T09:30:00+01:00 | 2024-05-10T10:00:00+01:00
0 | 2024-05-10T10:00:00+01:00 | 2024-05-10T10:30:00+01:00
0 | 2024-05-10T10:30:00+01:00 | 2024-05-10T11:00:00+01:00
0 | 2024-05-10T11:00:00+01:00 | 2024-05-10T11:30:00+01:00
0 | 2024-05-10T11:30:00+01:00 | 2024-05-10T12:00:00+01:00
0 | 2024-05-10T12:00:00+01:00 | 2024-05-10T12:30:00+01:00
0.029 | 2024-05-10T12:30:00+01:00 | 2024-05-10T13:00:00+01:00
0 | 2024-05-10T13:00:00+01:00 | 2024-05-10T13:30:00+01:00
0 | 2024-05-10T13:30:00+01:00 | 2024-05-10T14:00:00+01:00
0 | 2024-05-10T14:00:00+01:00 | 2024-05-10T14:30:00+01:00
0 | 2024-05-10T14:30:00+01:00 | 2024-05-10T15:00:00+01:00
0 | 2024-05-10T15:00:00+01:00 | 2024-05-10T15:30:00+01:00
0 | 2024-05-10T15:30:00+01:00 | 2024-05-10T16:00:00+01:00
0 | 2024-05-10T16:00:00+01:00 | 2024-05-10T16:30:00+01:00
0 | 2024-05-10T16:30:00+01:00 | 2024-05-10T17:00:00+01:00
0 | 2024-05-10T17:00:00+01:00 | 2024-05-10T17:30:00+01:00
0.031 | 2024-05-10T17:30:00+01:00 | 2024-05-10T18:00:00+01:00
0.065 | 2024-05-10T18:00:00+01:00 | 2024-05-10T18:30:00+01:00
0 | 2024-05-10T18:30:00+01:00 | 2024-05-10T19:00:00+01:00
0 | 2024-05-10T19:00:00+01:00 | 2024-05-10T19:30:00+01:00
0 | 2024-05-10T19:30:00+01:00 | 2024-05-10T20:00:00+01:00
0 | 2024-05-10T20:00:00+01:00 | 2024-05-10T20:30:00+01:00
0 | 2024-05-10T20:30:00+01:00 | 2024-05-10T21:00:00+01:00
0 | 2024-05-10T21:00:00+01:00 | 2024-05-10T21:30:00+01:00
0 | 2024-05-10T21:30:00+01:00 | 2024-05-10T22:00:00+01:00
0 | 2024-05-10T22:00:00+01:00 | 2024-05-10T22:30:00+01:00
0 | 2024-05-10T22:30:00+01:00 | 2024-05-10T23:00:00+01:00
0 | 2024-05-10T23:00:00+01:00 | 2024-05-10T23:30:00+01:00
0 | 2024-05-10T23:30:00+01:00 | 2024-05-11T00:00:00+01:00



</body>

</html>

**Reply by BottlecapDave** [reply, marked as answer]
Ignore my previous answer, lost context of original question. The current consumption sensor just reports what comes back from the OE API. I'm not personally seeing this with my setup. It could be the data got reconciled during the night. When it happens again, it would be best to check if it's present in charges attribute of the accumulative consumption sensor. If it doesn't then it's probably OE correcting the data after being captured, which there's not anything I can do about
